Dockerfile

```
@@ -0,0 +1,33 @@
+FROM python:3.9-slim
+
+# install apps dependencies
+RUN apt update && apt install -y \
+    git \
+    curl \
+    build-essential
```

```suggestion
    build-essential \
    && rm -rf /var/lib/apt/lists/*
```
https://askubuntu.com/a/1251250